sim/warlock/rotations.go

```diff
@@ -87,26 +87,26 @@ func (warlock *Warlock) tryUseGCD(sim *core.Simulation) {
 			nextBigCD = cdReadyAt
 		}
 	}
-	allCDs := []time.Duration{
+	rotationalLeeway := []time.Duration{
```

Comment:
I do not think it's a good idea to rename this right now, even though the original variable name does not match perfectly.  This gives much more "diffs"  in your commit.
Anyways, right now I'm also reforging the rotation logic, so this variable will be embedded in a new variable.

Reply:
I'll revert this change in my branch with the fix and update accordingly.